google/cloud/forseti/services/explain/service.py

```diff
@@ -59,10 +59,14 @@ def _determine_is_supported(self):
         Returns:
             bool: True if Explainer module can be used.
         """
-        root_resource_id = (
-            self.explainer.config.inventory_config.root_resource_id)
-        if 'organizations' in root_resource_id:
-            return True
+        try:
+            root_resource_id = (
+                self.explainer.config.inventory_config.root_resource_id)
```

Comment:
Do you think its better to check for the none case beforehand instead of raising the exception?
Something along the line of 
```
if not (self.explainer.config.inventory_config && self.explainer.config.inventory_config.root_resource_id):
  return False
```

Reply:
Good idea, done.